prov/rxd/src/rxd_av.c

```diff
@@ -332,13 +324,10 @@ static int rxd_av_close(struct fid *fid)
 	if (ret)
 		return ret;
 
-	ofi_rbmap_cleanup(&av->rbmap);
 	ret = ofi_av_close(&av->util_av);
```

Comment:
Before we close the av, we might need to iterate through all eps and remove each fi_addr from the fi_addr hash

Reply:
^This again

Reply:
updated the rxd_av_delete_hashtable function to include this.